worlds/stardew_valley/locations.py

```diff
@@ -216,7 +216,7 @@ def extend_fishsanity_locations(randomized_locations: List[LocationData], option
         fish_locations = [location_table[f"{prefix}{fish.name}"] for fish in active_fish]
         randomized_locations.extend(filter_disabled_locations(options, fish_locations))
     elif fishsanity == Fishsanity.option_exclude_legendaries:
-        fish_locations = [location_table[f"{prefix}{fish.name}"] for fish in active_fish if fish not in legendary_fish]
+        fish_locations = [location_table[f"{prefix}{fish.name}"] for fish in active_fish if not fish.legendary]
```

Comment:
On line 209 above, we also use the array legendary_fish, I think it should use the bool property like this one

Reply:
that is true